decidim-assemblies/app/permissions/decidim/assemblies/permissions.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Assemblies
+    class Permissions
```

Comment:
Shouldn't this inherit from a common interface that implements the required methods with a `NotImplementedError`?

Reply:
I want to wait until I add another permissions class to a component to see what logic can be extracted.